docs/language.md

```diff
@@ -56,17 +44,19 @@ A path statement provides access and validation rules for data stored at a given
       }
     }
 
+If a Type is not given, `Any` is assumed.
+
 Path statements can also include wildcards parts whose values can then be used
 within an expression as a variable parameter.
 
-    path /top/$wildcard/$id {
-      validate() {
+    path /top/$wildcard/$id is Type {
+      write() {
         return $wildcard < "Z" && $id > 7;
       }
     }
 
-In expressions, the value of 'this' is defined to be the top level object when used in
-a Type or path storage location.
+In expressions, the value of `this` is defined to be *new* value at the path location (i.e.
+if writing, it is the value at the location after the write would have been complete).
```

Comment:
Consider the following edit instead:
In expressions, the value of `this` is the either current value to `read()` or the new value to `write()`.